src/schemaToCSVTransforms.js

```diff
@@ -0,0 +1,40 @@
+import _ from 'lodash/fp'
+import { updateMany } from './futil'
+
+// Maps contexture schemas to tranforms for fast-csv
+export let schemaToCSVTransforms = (schema, {logger = _.noop, header = true, include } = {}) => {
+  let count = 0
+  let transformedHeaders = include && _.reduce((acc, k) => ({...acc, [k]: _.get(`${k}.label`, schema) || _.startCase(k) }), {}, include)
+
+  let headers = _.mapValues('label', schema)
+  return {
+    transformedHeaders,
+    transformHeaders: key => headers[key] || _.startCase(key),
+    // NOTE: for whatever reason you can't use a function declared
+    // by _.flow as transaform, just `transform: _.identity` works as expected
+    // but `transform: _.flow(_.identity)` does write a csv and doesn't throw any errors :(
+    // _.flow might be hitting the fast-csv callback api for transform but not really sure
+    transform: row => _.flow(
+      include ? _.pick(include) : _.identity,
+      _.cond([
```

Comment:
I think it will be more readable and cleaner to use F.ifElse e.g.
```javscript
 F.ifElse(
        header && count === 0,
        _.identity,
        _.flow(
          updateMany(_.mapValues('display', schema)),
        )
      ),

Reply:
I am not sure why do we use _.flow for 
          updateMany(_.mapValues('display', schema))?